source/user-manual/capabilities/vulnerability-detection/offline-update.rst

```diff
@@ -377,6 +377,6 @@ Or in a local path:
 
     <provider name="msu">
         <enabled>yes</enabled>
-        <path>/local_path/msu-updates.json.gz</path>
+        <path>/local_path/msu-updates\.json\.gz$</path>
```

Comment:
Same than before: if this is a single file, expressing the path as a regex that only has one possible match isn't required.